From 0d7c92bcebcbabceebdf5c0fa7768c9f92fbf9f7 Mon Sep 17 00:00:00 2001
From: Gent8 <deboordermounir@gmail.com>
Date: Sun, 19 Oct 2025 16:42:32 +0200
Subject: [PATCH 22/24] Update plural form for "one" in Amharic localization in
 plurals.xml to include dynamic time formatting for improved language support

---
 i18n/src/commonMain/moko-resources/am/plurals.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/i18n/src/commonMain/moko-resources/am/plurals.xml b/i18n/src/commonMain/moko-resources/am/plurals.xml
index 1dc35ec91..e14f5435f 100644
--- a/i18n/src/commonMain/moko-resources/am/plurals.xml
+++ b/i18n/src/commonMain/moko-resources/am/plurals.xml
@@ -6,7 +6,7 @@
         <item quantity="other">%d ምድቦች</item>
     </plurals>
     <plurals name="lock_after_mins">
-        <item quantity="one">ከ 1 ደቂቃ በኋላ</item>
+        <item quantity="one">ከ %1$s ደቂቃ በኋላ</item>
         <item quantity="many">ከ %1$s ደቂቃዎች በኋላ</item>
         <item quantity="other">ከ %1$s ደቂቃዎች በኋላ</item>
     </plurals>
-- 
2.39.5 (Apple Git-154)

